src/js/common/components/AlertBox.jsx

```diff
@@ -33,7 +33,7 @@ class AlertBox extends React.Component {
 
   render() {
     if (!this.props.isVisible) {
-      return <div aria-live="assertive"/>;
+      return null;
```

Comment:
Hopefully doesn't affect other apps. Just seems more proper that it should not display something that shouldn't be displayed rather than insert an empty `div`. Even with this, I think if an alert appears, it should still alert the screen reader, right?